pr_testing/get_external_name.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -ex
+# gets repo name, then returns spec name depending on the rule
+PKG_REPO=$1
+PKG_NAME=$(echo ${PKG_REPO} | sed 's|.*/||')
+ARRAY=(
+#   Array of packages to keep track of
+#   Needs to be all lowercase
+#   "REPO_NAME:SPECK_NAME"
+    "cms-externals/llvm-project:llvm"
+    "cms-sw/scram:SCRAMV1"
```

Comment:
@gudrutis , repo name is cms-sw/SCRAM (at least whats what we get in the web hooks).